### ISSUE TITLE
list of pure ips as output

### ISSUE THREAD

**Issue by AhmedBakrDA** [subfission/HostResolver#1]
Hi, thanks for your awesome tool;
can you modify it to output a pure list of ip addresses to use it with another tools?

**Comment by subfission** [comment]
Absolutely, but what is the purpose since that is easily done with other tools?

**Comment by Anon-Exploiter** [comment]
+1 -- Need the IPs list.

**Comment by redactedgg** [comment]
/resolv.py hosts|cut -d"|" -f3 ?

**Comment by Anon-Exploiter** [comment]
It's easy to get the ips but should be supported officially by the tool using some argument.
